apollo-router/src/state_machine.rs

```diff
@@ -227,7 +227,7 @@ where
                 }
             };
 
-            tracing::info!("transitioned to {}", &new_state);
+            tracing::trace!("transitioned to {}", &new_state);
```

Comment:
@o0Ignition0o seemed okay with `debug` in his comment on the original issue.  I considered it.

My claim would be that it's better to make things more verbose if necessary and bury as much extra distraction as we can unless it's proven consistently necessary to make it louder at less-noisy levels.